SpatialGDK/Source/SpatialGDK/Public/Interop/WellKnownEntitySystem.h

```diff
@@ -28,10 +32,11 @@ class WellKnownEntitySystem
 
 	void InitializeVirtualWorkerTranslationManager();
 	void MaybeClaimSnapshotPartition();
+	void RetryServerWorkerEntityCreation(Worker_EntityId EntityId, int AttemptCounter);
 
 	const FSubView* SubView;
 
-	USpatialReceiver* Receiver;
+	USpatialNetDriver* NetDriver;
```

Comment:
Do we generally want to use `TWeakObjectPtr` to see if an object has been destroyed? I haven't seen many such usages in SpatialGDK code.

Reply:
Depends on the use case. For this one, the well known entity system has a well-defined lifecycle wrt the objects these members are pointing at, so I think a raw pointer here makes sense. 

To answer the actual question, I'm not sure there are any general rules we follow in the GDK for this kind of thing, unfortunately.

Reply:
Makes sense.